windows-agent/internal/cloudinit/cloudinit_test.go

```diff
@@ -172,6 +191,17 @@ hostagent_uid = landscapeUID1234
 				return
 			}
 
+			if tc.breakRemovingFile {
+				require.DirExists(t, path, "There should be a directory instead of agent.yaml")
+				return
+			}
+
+			if tc.skipProToken && tc.skipLandscapeConf {
+				// empty file should not exist
+				require.NoFileExists(t, path, "There should not be cloud-init agent file without useful contents")
```

Comment:
Since the `LoadWith*` family of functions all `require.NoError` when opening the file, I guess the way to do this is by first calling `Path` and checking if the returned path exists.

Reply:
makes sense to me!